test/check-application

```diff
@@ -15,37 +15,64 @@ import testlib
 
 class TestApplication(testlib.MachineCase):
     def testBasic(self):
-        b = self.browser
-        m = self.machine
+        """
+        Tests if changing the language works as expected.
+        """
```

Comment:
I deliberately did not split the test, as each new test has a considerable setup cost. But I'm okay with doing it as a way of demonstrating how to organize larger test suites. But then let's also rename this to `testTranslation`.